### ISSUE TITLE
Generate component with cli make the style file with css

### ISSUE THREAD

**Issue by OmarioHasan** [angular/angular#36617]
# 🐞 bug report

### Affected Package
The issue is caused by package @angular/cli

 Yes, the previous version in which this bug was not present was: 8.x.x

### Description
i'm making my local and global schematics generating scss files but it's not working it generation css files for components i know i can use --style=scss but still this a problem and thank you

## 🌍  Your Environment

Angular CLI: 9.1.1
Node: 12.14.1
OS: win32 x64

Angular: 9.0.0
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker
Ivy Workspace: Yes

`Angular CLI: 9.1.1
Node: 12.14.1
OS: win32 x64

Angular: 9.0.0
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.1
@angular-devkit/build-angular     0.900.1
@angular-devkit/build-optimizer   0.900.1
@angular-devkit/build-webpack     0.900.1
@angular-devkit/core              9.0.1
@angular-devkit/schematics        9.1.1
@angular/cli                      9.1.1
@angular/fire                     5.4.2
@ngtools/webpack                  9.0.1
@schematics/angular               9.1.1
@schematics/update                0.901.1
rxjs                              6.5.4
typescript                        3.7.5
webpack                           4.41.2`

![Annotation 2020-04-14 142225](https://user-images.githubusercontent.com/8146344/79224446-616eda80-7e5b-11ea-8c93-6ae632ecc931.png)
![image](https://user-images.githubusercontent.com/8146344/79224472-6cc20600-7e5b-11ea-8b70-69c959fb06a8.png)

**Comment by geromegrignon** [comment]
Please add a repo link to your project in order to test it directly.

**Comment by OmarioHasan** [comment]
@geromegrignon  for sorry it's private project in my company 😔

**Comment by geromegrignon** [comment]
could you try to make a new project based on you angular cli version to test ?

**Comment by OmarioHasan** [comment]
Yes sure that's a good idea I will make it and share this repo with you

**Comment by ericmartinezr** [comment]
This should go in cli's repo https://github.com/angular/angular-cli :+1:
